controllers/error.controller.js

```diff
@@ -1,3 +1,9 @@
 exports.errorHandler = (err, req, res, next) => {
+    if(err.status && err.msg){
+        res.status(err.status).send({msg: err.msg})
+    } else if (err.code === "22P02"){
```

Comment:
Nice! As the number of PSQL errors increases, you may want to separate this into its own error handling function (e.g. handlePSQLErrors) 👍